_pytest/runner.py

```diff
@@ -494,9 +494,12 @@ def importorskip(modname, minversion=None):
     """
     __tracebackhide__ = True
     compile(modname, '', 'eval') # to catch syntaxerrors
+    should_skip = False
     try:
         __import__(modname)
     except ImportError:
+        should_skip = True
+    if should_skip:
```

Comment:
make a issue on the issue tracker and add a link inside a comment of the fix,
then we can sort it out soon